core/src/test/scala/unit/kafka/admin/ReassignPartitionsCommandArgsTest.scala

```diff
@@ -230,15 +214,11 @@ class ReassignPartitionsCommandArgsTest {
   }
 
   @Test
-  def testInvalidCommandConfigArgumentForLegacyGenerate(): Unit = {
-    val args = Array(
-      "--zookeeper", "localhost:1234",
-      "--generate",
-      "--broker-list", "101,102",
-      "--topics-to-move-json-file", "myfile.json",
-      "--command-config", "/tmp/command-config.properties"
-    )
-    shouldFailWith("You must specify --bootstrap-server when using \"[command-config]\"", args)
+  def shouldPrintHelpTextIfHelpArg(): Unit = {
+    val args: Array[String]= Array(
+      "--help",
+      "--bootstrap-server", "localhost:1234")
```

Comment:
Hmm, we should not require this line to see the help text.

Reply:
Well, I agree. Making the `--bootstrap-server` as required argument caused this side effect. I think we should keep the `--bootstrap-server` as non-required, and do the argument check by ourselves. I'll update it, and also the other PR: https://github.com/apache/kafka/pull/10457 on the `ConfigCommand` tomorrow. Thank you.

Reply:
Makes sense.